Tchap/Modules/Room/RoomViewController.m

```diff
@@ -2581,6 +2581,60 @@ - (void)showAdditionalActionsMenuForEvent:(MXEvent*)selectedEvent inCell:(id<MXK
         }
 #endif
         
+        NSString *titleKey;
+        if (![self.roomDataSource.room.accountData getTaggedEventInfo:selectedEvent.eventId withTag:kMXRoomTagFavourite])
+        {
+            titleKey = @"room_event_action_favourite";
+        }
+        else
+        {
+            titleKey = @"room_event_action_remove_favourite";
+        }
+        [currentAlert addAction:[UIAlertAction actionWithTitle: NSLocalizedStringFromTable(titleKey, @"Vector", nil)
+          style:UIAlertActionStyleDefault
+        handler:^(UIAlertAction * action) {
+            
+            if (weakSelf)
+            {
+                typeof(self) self = weakSelf;
+                
+                [self cancelEventSelection];
+                
+                [self startActivityIndicator];
+                
+                if ([titleKey isEqual:@"room_event_action_favourite"])
```

Comment:
you should use `isEqualToString` instead of `isEqual` as soon as the object is a string